src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Menu, Dropdown } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import { NavLink } from 'react-router-dom';
+import { Avatar } from 'antd';
+
+import { UserOutlined } from '@ant-design/icons';
+
+interface DropDownProps {}
+
+const menu = (
+  <Menu>
+    <Menu.Item key="0">
+      <NavLink className="activeNot" to="/login">
+        Log in
+      </NavLink>
+    </Menu.Item>
+    <Menu.Item key="1">
+      <a href="#">2nd menu item</a>
+    </Menu.Item>
+    <Menu.Divider />
+    <Menu.Item key="3">Log out</Menu.Item>
+  </Menu>
+);
+
+//nu mai pot inchide dropdownul.
+
+const DropDown: React.FC<DropDownProps> = ({}) => {
+  return (
+    <div>
```

Comment:
do you need a wrapper div? if you are using this for layout purposes you could use items from antd layout(grid etc)